src/pyramid/view.py

```diff
@@ -201,6 +201,16 @@ def my_view(context, request):
         ``view_config`` will work ONLY on module top level members
         because of the limitation of ``venusian.Scanner.scan``.
 
+        Also it will ONLY work on views present as plaintext python
+        files (*.py), ignoring all non-*.py ones which also includes
+        *.pyc and *.pyo (python precompiled byte code / optimized) files.
+
+        Another venusian specific precondition to take into account is that
+        python packages to be found by venusian need to have a __init__.py
+        file in their root directory.
```

Comment:
This is unfortunately an issue with Python packages. Packages that don't contain a `__init__.py` are considered namespace packages, and the default tools to walk a package/module don't actually return namespace packages.

See this issue for further clarification:
https://github.com/Pylons/venusian/issues/68